cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java

```diff
@@ -171,9 +171,14 @@ protected ApplicationManager deployApplication(Class<? extends Application> appl
    *
    * @deprecated Use {@link TestManager#clear()} from {@link #getTestManager()}.
    */
-  protected void clear() throws Exception {
-    TestManager testManager = getTestManager();
-    testManager.clear();
+  protected void clear() {
+    try {
+      TestManager testManager = getTestManager();
+      testManager.clear();
+    } catch (Exception e) {
+      // Unchecked exception to maintain compatibility until we remove this method
+      Throwables.propagate(e);
```

Comment:
It seems like the propagate method already throws it:
throw new RuntimeException(throwable);